### PR TITLE
Update CefSharp to latest version

### PR DESCRIPTION
**Pull request by lnatella**
Fixes [issue-number] 
   - e.g Fixes #2345

**Summary:** [summary of the change and which issue is fixed here]
   - e.g. I have added a new feature to the Javascript Binding implementation

**Changes:** [specify the structures changed] 
   - e.g. I have modified the Javascript Bindign Implementation
      - Added support for Async binding to return Task<T>
      - Added new QUnit Test cases
      
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Updated documentation

**Checklist:**
- [ ] Tested the code(if applicable)
- [ ] Commented my code
- [ ] Changed the documentation(if applicable)
- [ ] New files have a license disclaimer
- [ ] The formatting is consistent with the project (project supports .editorconfig)

### CODE REVIEW

File: CefSharp.Example/CefExample.cs [hunk 1]
```diff
@@ -188,7 +193,7 @@ public static void Init(AbstractCefSettings settings, IBrowserProcessHandler bro
             settings.RegisterScheme(new CefCustomScheme
             {
                 SchemeName = "localfolder",
-                SchemeHandlerFactory = new FolderSchemeHandlerFactory(rootFolder: DebuggingSubProcess ? @"..\..\..\..\CefSharp.Example\Resources" : "Resources",
```
Comment (mol) [inline CefSharp.Example/CefExample.cs]: Isn't this one of our changes? I wonder if it's necessary to keep this for some kind of debugging? But if not then this is ok.

File: CefSharp.Wpf.Example/CefSharp.Wpf.Example.csproj [hunk 2]
```diff
@@ -246,7 +246,4 @@
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <Import Project="..\CefSharp.props" />
   <Import Project="..\CefSharp.Native.props" />
-  <PropertyGroup>
```
Comment (mol) [inline CefSharp.Wpf.Example/CefSharp.Wpf.Example.csproj:249]: I believe this is necessary in order to run the WPF example project from within the Mailbird solution. Try cleaning your solution and see if you can get it running without this (to confirm this isn't necessary).

**Review by mol — CHANGES_REQUESTED** [review]
I scrolled through all changes and added a couple of comments in places where I'd like you to test that the code isn't breaking - because I recall those changes as having been made by us (if I remember correctly). Other than that everything looks ok.

File: CefSharp.Example/CefExample.cs (hunk 1 above)
Comment (lnatella) [reply CefSharp.Example/CefExample.cs]: I rolled back this change, better to keep it as it was

File: CefSharp.Wpf.Example/CefSharp.Wpf.Example.csproj (hunk 2 above)
Comment (lnatella) [reply CefSharp.Wpf.Example/CefSharp.Wpf.Example.csproj:249]: Looks like they moved these resources online, for example https://cefsharp.example/BindingTest.html

**Review by mol — APPROVED** [review]
This is ok to merge now then assuming we still want to if we want to remove WPF.

**Pull request merged**